editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/AdvancedEndpointPropertiesSection.java

```diff
@@ -147,6 +146,9 @@ protected void generateTabContents(List<Parameter> props, final Composite page,
         		continue;
         	}
 
+        	// we currently don't want to display class params of type element
+        	if (CamelComponentUtils.isClassProperty(p) && p.getKind().equalsIgnoreCase(AbstractCamelModelElement.NODE_KIND_ELEMENT)) continue;
```

Comment:
For Another Review: we should extract method which is checking if it needs to be displayed or not, we would be able to factorise this kind of code and make it clearer.
(hard to understand at first read as it is in a long method with continue at end of lines)